front/components/assistant_builder/actions/WebsearchAction.tsx

```diff
@@ -13,7 +13,7 @@ export function ActionWebsearch() {
   return (
     <div>
       This action will perform a web search and return the top results (title,
-      link and summary) to the assistant.
+      link and summary, and page content) to the assistant.
```

Comment:
same here adapt wording
I think there is a product question here; just browsing a given url is a legit use case, so we should make it clear to users -- not in the direct scope of this PR but ccing @Duncid and @spolu who were part of this discussion
E.g. I am on a website, don't want to read it, copy paste the url to the assistant and tell him to read it for me
=> user has to understand that this action covers the use case, so we need the right copy here

Reply:
and also @gabhubert who is copy master right?